### ISSUE TITLE
IDE Reporter: optional config parameter to record not only survived but also killed mutations

### ISSUE THREAD

**Issue by stanislaw** [mull-project/mull#592]
This is how the output with no survived mutants looks like:

```
Survived mutants (0/1):

Mutation score: 100%
```

Would be great to have a command-line option that prints killed mutants.

This is how it looks like when the mutants have survived:

```
Survived mutants (1/1):

/sandbox/mull/tests-lit/tests/mutations/math/cxx_add_to_sub/02_basic/sample.cpp:11:12: warning: Replaced + with -
  return a + b;
           ^
Mutation score: 0%
```

**Comment by stanislaw** [comment]
We could have something like this:

```
Killed mutants (1/2):

/sandbox/mull/tests-lit/tests/mutations/math/cxx_add_to_sub/02_basic/sample.cpp:11:12: warning: Replaced + with -
  return a + b;
             ^

Survived mutants (1/2):

/sandbox/mull/tests-lit/tests/mutations/math/cxx_add_to_sub/02_basic/sample.cpp:11:12: warning: Replaced + with -
  return a + b;
           ^

Mutation score: 50%
```

**Comment by stanislaw** [comment]
Also the option itself: should it be `-ide-reporter-show-killed`?

**Comment by stanislaw** [comment]
Closed by #640.
